doc/classes/DisplayServer.xml

```diff
@@ -48,11 +51,13 @@
 			<param index="2" name="flags" type="int" />
 			<param index="3" name="rect" type="Rect2i" default="Rect2i(0, 0, 0, 0)" />
 			<description>
+				Create a new subwindow (a [Window] whose presence is linked to another window). This window will appear to be embedded within the main window if [member Viewport.gui_embed_subwindows] is [code]true[/code]. Otherwise, the window will appear to be a separate window which can be dragged outside the main window. [code]vsync_mode[/code] is ignored if [member Viewport.gui_embed_subwindows] is [code]true[/code], as the subwindow will be drawn at the same time as the main window in this case.
```

Comment:
`DisplayServer` will always create a native window, it won't show it (and you won't be able to do it manually, since `show_window` is not exposed for some reason). And the new window will not have any `Viewport` or `Window` object associated with it until `window_attach_instance_id` is called, so the window will not render anything as well (e.g. sold pink background in case of macOS).

Reply:
I guess these should not be accessible at all, it's pretty useless methods and only intended to be used by `Window` - https://github.com/godotengine/godot/pull/67933